src/pair.py

```diff
@@ -10,6 +10,79 @@
 from hoomd.md import pair
 
 
+class Colloid(pair.Pair):
+    r"""Colloid pair potential.
+
+    Args:
+        nlist (hoomd.md.nlist.NeighborList): Neighbor list.
+        r_cut (float): Default cutoff radius :math:`[\mathrm{length}]`.
+        mode (str): Energy shifting/smoothing mode.
+
+    :py:class:`Colloid` is the effective Lennard-Jones potential obtained by
+    integrating the Lennard-Jones potential between a point and a sphere or a
+    sphere and a sphere. The attractive part of the colloid-colloid pair
+    potential was derived originally by Hamaker, and the full potential by
+    `Everaers and Ejtehadi <http://doi.org/10.1103/PhysRevE.67.041710>`_.
+    A discussion of the application of these potentials to colloidal suspensions
+    can be found in `Grest et al. <http://dx.doi.org/10.1063/1.3578181>`_
+
+    The pair potential has three different coupling styles between particle types:
+
+    - Solvent-Solvent gives the Lennard-Jones potential for coupling between
+    pointlike particles
+
+    .. math::
+        :nowrap:
+
+        \begin{eqnarray*}
+        V(r)  = & \frac{A}{36} \left[\left(\frac{\sigma}{r}\right)^{12}
+                  - \left(\frac{\sigma}{r}\right)^6 \right] & r < r_{\mathrm{cut}} \\
+              = & 0 & r \ge r_{\mathrm{cut}}
+        \end{eqnarray*}
+
+    - Colloid-Solvent gives the interaction between a pointlike particle and a colloid
+    - Colloid-Colloid gives the interaction between two colloids
+
+    Refer to the work by `Grest et al. <http://dx.doi.org/10.1063/1.3578181>`_ for the
+    form of the colloid-solvent and colloid-colloid potentials, which are too cumbersome
+    to report on here.
```

Comment:
I think we can make these comments a little more helpful in two ways:
1. Describe the cases using the radii (rather than colloid-solvent and colloid-colloid)
2. Can we cite equation numbers from the Grest paper that the reader can refer to? I think you can stick the url in one place and refer to it multiple times, like here: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages .


It's Eqs. (3) & (4) for the colloid-colloid potential, and Eq. (5) for the colloid-solvent.